package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@datadog/datadog-ci",
-  "version": "2.45.0",
+  "version": "2.46.0",
```

Comment:
This won't be a breaking change, so we might want to bump the patch version only?

Reply:
@tanishq-dubey can you undo this change in your PR? Bumping the version in `package.json` is part of the release process and should be done in its own PR